src/oss/langgraph/test.mdx

```diff
@@ -1,3 +1,317 @@
 ---
-title: (Coming Soon) Test
+title: Test
 ---
+
+import AlphaCallout from '/snippets/alpha-lc-callout.mdx';
+
+<AlphaCallout />
+
+This guide covers more advanced scenarios around lower-level graphs you've created with custom structures - if you are just getting started, check out [this section that uses LangChain prebuilt agents instead](/langchain/test/).
```

Comment:
`/langchain/test` didn't seem to go to the right place locally, I am not sure what the right scheme is here